src/testdrive/src/parser.rs

```diff
@@ -348,10 +378,9 @@ impl<'a> Iterator for LineReader<'a> {
                     continue;
                 } else if line.chars().all(char::is_whitespace) {
                     line.clear();
-                    fold_newlines = should_fold(chars.peek().map(|c| c.1));
```

Comment:
@benesch I removed the "continue" below so that empty lines are produced out of this method, but then the compiler told me the ```fold_newlines``` needs to be removed. I must admit I do not understand that bit of code so this change is suspicious